### PR TITLE
[AC-4389] Recompile Ti.Flurry to support Android 7.0.

### PR DESCRIPTION
**Pull request by rodperottoni**
Recompiled Ti.Flurry to support Android 7.0

### CODE REVIEW

**Comment by ashcoding** [comment]
2 more things you need to do.

File: android/manifest [hunk 1]
```diff
@@ -2,7 +2,7 @@
 # this is your module manifest and used by Titanium
 # during compilation, packaging, distribution, etc.
 #
-version: 1.3.1
+version: 1.3.2
 apiversion: 2
```
Comment (ashcoding) [inline android/manifest]: apiversion is 3

File: android/manifest [hunk 2]
```diff
@@ -2,7 +2,7 @@
...
 architectures: armeabi armeabi-v7a x86
```
Comment (ashcoding) [inline android/manifest]: Refer to https://github.com/appcelerator-modules/ti.map/blob/2f1b7591a7a46e5d4379727f4d3fdbb0168b90fb/android/manifest#L6-L7 and remove ```armeabi```

File: android/manifest (hunk 1 above)
Comment (rodperottoni) [inline android/manifest]: what does this affect?

File: android/manifest (hunk 2 above)
Comment (rodperottoni) [inline android/manifest]: what does this affect?

File: android/manifest (hunk 1 above)
Comment (ashcoding) [inline android/manifest]: Anything that's compiled with TiSDK 6+ would need a bump to apiversion 3. This is to differentiate them. TiSDK 6+ has an updated V8 for Android. They are totally not compatible with any modules compiled prior to TiSDK 6.

File: android/manifest (hunk 2 above)
Comment (ashcoding) [inline android/manifest]: With the new V8 in TiSDK6 armeabi isn't needed.

**Comment by ashcoding** [comment]
Also, please remove `android/dist/flurry_android.jar`

**Comment by ashcoding** [comment]
You could instead include the `ti.flurry.zip` here.

**Comment by rodperottoni** [comment]
Here you go: 

[ti.flurry-android-1.3.2.zip](https://github.com/appcelerator-archive/ti.flurry/files/436322/ti.flurry-android-1.3.2.zip)

**Comment by hansemannn** [comment]
@rodperottoni Please bump to 2.0.0, since it is a major change. We do the major bump for all V8-updates. If you want help porting them, we would greatly appreciate that!

**Comment by rodperottoni** [comment]
Will do.

**Comment by rodperottoni** [comment]
Completely forgot about this until I had to use it on a new project here. Bumped to 2.0.0.
[ti.flurry-android-2.0.0.zip](https://github.com/appcelerator-archive/ti.flurry/files/560835/ti.flurry-android-2.0.0.zip)

**Comment by hansemannn** [comment]
Thanks man! And I guess it works fine for you. Will merge then! 🚀

**Pull request merged**